test/common/apiHelpers.js

```diff
@@ -154,6 +154,13 @@ function getBalance (address, cb) {
 	http.get('/api/accounts/getBalance?address=' + address, httpCallbackHelper.bind(null, cb));
 }
 
+function getBlocks (params, cb) {
+	var url = '/api/blocks';
+	url = paramsHelper(url, params);
+
+	http.get(url, httpCallbackHelper.bind(null, cb));
```

Comment:
Rather than using httpCallbackHelper, we should use something like httpCallbackWithStatusHelper which returns `res.body` & `res.status`. And we should check for status code for each test

Reply:
I will change the promise functions then so they return the whole `res` object.